### PR TITLE
Standardise distance calculations.

### PR DESCRIPTION
**Pull request by cydine**
## Description

Ditched geopy in favour of local functions to improve performance. Removed duplicate functions. Standardised earth radius for all functions.
## Motivation and Context

Several different formulas and constants were being used to perform the same distance calculations - not ideal. Using Vincenty is over the top for the distances we are dealing with.
## How Has This Been Tested?

Tested on my own machine. Pretty sure I've not broken anything......
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pogom/models.py [hunk 1]
```diff
@@ -378,15 +377,13 @@ def get_spawnpoints_in_hex(cls, center, steps):
 
         s = list(query.dicts())
 
-        # The distance between scan circles of radius 70 in a hex is 121.2436
-        # steps - 1 to account for the center circle then add 70 for the edge.
-        step_distance = ((steps - 1) * 121.2436) + 70
+        # The distance between scan circles of radius 70 in a hex is 120
+        step_distance = ((steps * 120.0) - 50.0) / 1000.0
```
Comment (averylarry) [inline pogom/models.py]: Why did you change this?

**Comment by averylarry** [comment]
Note that you also changed the gym info radius from 1km to 450m.

**Comment by springjools** [comment]
Went throught the code and changed the formula to the haversine formula, transform.py still needs to be added back

**Comment by springjools** [comment]
It is unclear what the status is currently, it needs rebasing but is this still a topical issue?

**Comment by sebastienvercammen** [comment]
This is an old PR that's no longer maintained. If this is still relevant, it should be rebased and we'll reopen if the author replies.
